presentation/src/main/java/utils/OutlinedTextView.kt

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import com.d101.presentation.R
+
+class OutlinedTextView
+@JvmOverloads
+constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+) : androidx.appcompat.widget.AppCompatTextView(context, attrs, defStyleAttr) {
+    private var strokeColor: Int
+    private var strokeWidthVal: Float
+
+    init {
+        // attributes 가져오기
+        val typedArray = context.obtainStyledAttributes(attrs, R.styleable.OutLineTextView)
+        strokeWidthVal = typedArray.getFloat(R.styleable.OutLineTextView_textStrokeWidth, 3f)
+        strokeColor = typedArray.getColor(R.styleable.OutLineTextView_textStrokeColor, Color.WHITE)
+        typedArray.recycle()
+    }
+
+    override fun onDraw(canvas: Canvas) {
+        // draw stroke
+        val states: ColorStateList = textColors
+        paint.style = Paint.Style.STROKE
+        paint.strokeWidth = strokeWidthVal
+        setTextColor(strokeColor)
```

Comment:
🚫 **[ktlint]** <standard:indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (8) (should be 12)